scripts/update/Updater.php

```diff
@@ -164,5 +165,13 @@ public function update($initialVersion) {
         }
 
         $this->skip('6.1.0', '6.1.2');
+
+        if ($this->isVersion('6.1.2')) {
+            $this->getServiceManager()->register(RdsResultStorage::SERVICE_ID, new RdsResultStorage([
+                RdsResultStorage::OPTION_PERSISTENCE => 'default',
+                RdsResultStorage::OPTION_COMPATIBLE_SCHEMA => new RdsCompatibleSchema(),
```

Comment:
serialization of RdsCompatibleSchema will be unclean

Reply:
Fixed.